composer.json

```diff
@@ -8,12 +8,12 @@
     "require": {
         "php": ">=7.4",
         "ext-json": "*",
-        "symfony/http-kernel": "^5.0 || ^6.0",
```

Comment:
Since we dont officially support v6 yet in our systems, it makes sense to me not to do that here yet, either. :+1: